test/batch_job.jl

```diff
@@ -3,7 +3,7 @@
 
     @testset "status_reason" begin
         @testset "not provided" begin
-            patch = @patch function describe_jobs(; kwargs...)
+            patch = @patch function AWSBatch.Batch.describe_jobs(args...; kwargs...)
```

Comment:
Do these need to be `AWSBatch.Batch.describe_jobs()`? Would `AWSBatch.describe_jobs()` not suffice?

Reply:
Yes, it needs to be `AWSBatch.Batch.describe_jobs`, it needs to patch the function in `Batch`, the function in `AWSBatch` is different.